src/wwwroot/app/botConfig.js

```diff
@@ -62,4 +70,35 @@ export default function _config(
 
    LoggerServiceProvider.changeLogLevel(context.logLevel);
    HttpServiceProvider.changeApiUrl(context.serverUrl);
+
```

Comment:
Please, create `validationConfig`. and replace all cofigs for validation into this file